src/azul/service/manifest_controller.py

```diff
@@ -137,28 +137,35 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            command_lines = self.service.command_lines(manifest, str(location))
             body = {
                 'Status': 301,
                 'Location': location.url,
                 'Retry-After': token.wait_time
             }
         else:
-            if fetch:
+            try:
+                query_params['objectKey']
```

Comment:
If you don't need the value, then you shouldn't use EAFP. You should just `if k in d`.

However it is not obvious why `if fetch` was changed to `if 'objectKey' in query_params`

You can try adding a comment but you risk that I won't understand your comment. So you probably want to explain this to me during PL. Up to you.

Reply:
Revert to using `fetch` as the flag.